src/config.cr

```diff
@@ -17,9 +17,18 @@ require "./constants"
 log_level = PlaceOS::Core.production? ? ::Log::Severity::Info : ::Log::Severity::Debug
 log_backend = PlaceOS::LogBackend.log_backend
 
+namespaces = ["action-controller.*", "place_os.*"]
```

Comment:
don't we usually use "placeos" instead of "place_os"

Reply:
nearly everywhere, i've started just using `Log.for(self)`, which for `PlaceOS` produces `"place_os"`

Reply:
i think `Log.for(self)` is the best way to go, makes things more impervious to updates to symbols